activities/staff-information/sickness.md

```diff
@@ -45,3 +45,19 @@ This enables us to know you are back in and avoids action by the arbo-dienst.
 
 The operations coordinator immediately reports the sick leave in [the portal of our absenteeism insurance](https://mijnwerkgeversportaal.acumen.nl/index3.html).
 As soon as the employee is better it needs to be reported in the same portal. This can also be done retroactively if the operations coordinator is not present on the day the employee goes back to work.
+
+### Long-term sick leave management
+
+In case an employee is sick for a longer time [an ARBO (company) doctor will be involved](https://www.arboned.nl/en/absence-support/company-doctor-as-specialist). Our ARBO service is [RegioPoortwachters](https://www.regiopoortwachters.nl/). A case manager from RegioPoortwachters will reach out to inquire about the sick employee within approx. 10 days after the initial sickness report in the insurance portal. The ARBO service portal is integrated with our insurance portal.
+
+A meeting with an ARBO doctor will be planned to assess the situation of the employee. Depending on the outcome of the assessment, reintegration will be initialized when the situation allows. [A reintegration plan (plan van aanpak) will be drafted for employees who get ill](https://business.gov.nl/regulation/working-conditions-employees/). This plan should include the procedures all involved parties must follow. The director must approve the plan.
+
+### Reintegration management
+
+The case manager is the main contact person between the sick employee and the director. Updates are send regularly. It's important that the percentage of sickness vs reintegration hours are tracked in the insurance portal. E.g. if an employee is working 20% this means they are sick for 80%. This needs to be kept up-to-date as the insurance pay-out is based on the percentage of sickness.
```

Comment:
Is the director the chief executive, or a member of the board of directors? (Update everywhere for clarity.)

Reply:
Separately, is there any role for HR in this process?

Reply:
The director is the chief executive. Will review and update accordingly. Yes, it was my job to do but since Boris will take over these tasks and no replacement will be hired I wrote it as such to reflect the nearly current situation.